apps/web/src/state/farmsV3/hooks.ts

```diff
@@ -117,7 +117,7 @@ export const useFarmsV3 = ({ mockApr = false, boosterLiquidityX = {} }: UseFarms
   const cakePrice = useCakePrice()
 
   const { data } = useQuery({
-    queryKey: [chainId, 'cake-apr-tvl'],
+    queryKey: [chainId, 'cake-apr-tvl', Object.keys(boosterLiquidityX).join('-')],
```

Comment:
i think it's ok to simply use `boosterLiquidityX` as key since the queryKey is actually hashed